detection_rules/etc/version.lock.json

```diff
@@ -4306,9 +4383,30 @@
       }
     },
     "rule_name": "Account Configured with Never-Expiring Password",
-    "sha256": "323d4cd6580d5345a3d47924597c0d860fb1dba813e9aef86cf76e4558a03349",
+    "sha256": "5bc3884f85cb23ef9dcb68da3accd2c174e888f14003dd0120889a9e973072ed",
     "type": "query",
-    "version": 100
+    "version": 101
+  },
+  "63c05204-339a-11ed-a261-0242ac120002": {
+    "min_stack_version": "8.4",
+    "rule_name": "Kubernetes Suspicious Assignment of Controller Service Account",
+    "sha256": "6ff3774856f3a89c719426d0b0ad31e9476927c25e33ae9f1fb2c33a60262fe9",
+    "type": "query",
+    "version": 1
+  },
+  "63c056a0-339a-11ed-a261-0242ac120002": {
+    "min_stack_version": "8.4",
+    "rule_name": "Kubernetes Denied Service Account Request",
+    "sha256": "206708bf073f5373d61076aff081083306733334434485a072a95127453c17f3",
+    "type": "query",
+    "version": 1
+  },
+  "63c057cc-339a-11ed-a261-0242ac120002": {
+    "min_stack_version": "8.4",
+    "rule_name": "Kubernetes Anonymous Request Authorized",
+    "sha256": "a2254efd45c81509a0821dfe2637b59d41176b00bcf5e992e9667ec09a5053d0",
+    "type": "query",
+    "version": 1
```

Comment:
Confirmed these are the K8s rules that we had to change minimum stack version and backport to 8.4 branch.